doc/release-notes/0.32/0.32.md

```diff
@@ -69,6 +69,12 @@ The following table covers notable changes in v0.32.0. Further information about
 <td valign="top">Build for Apple Silicon (AArch64) macOS is available as an early access release.  It is stable enough for evaluation but not suitable for production yet.</td>
 </tr>
 
+<tr><td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/14424">#13492</a></td>
```

Comment:
The link (14424) doesn't match the text (13492)

Reply:
Good catch. Fixed.

Reply:
I think 13492 was correct, not 14424.
https://github.com/eclipse-openj9/openj9/pull/13492 - Remove JITServer tech preview message from the server
https://github.com/eclipse-openj9/openj9/pull/14424 - Don't set remoteCompReq flag if doing an AOT load

Reply:
My interpretation is that the issue that needs to be mentioned is the one that facilitated the update of the documentation. In this case it was the bug fix in #14424 which allowed JITServer to run successfully on Z. If my interpretation is incorrect, I'll change the text to link to #13492.

Reply:
It's not wrong but there is no obvious connection between 13492 and JIT server. The commit comment doesn't explain why the change is made, what it affects, or link to any issue. Although other issues link to it. I'll go ahead and merge this as-is. We have this conversion in the PR to explain it.